helm/dex-app/templates/dex/deployment.yaml

```diff
@@ -20,6 +20,15 @@ spec:
         {{- include "dex.labels.common" . | nindent 8 }}
       annotations:
     spec:
+      affinity:
```

Comment:
This affinity term means that we try to schedule each replica on a separate node if possible.

Reply:
Okie, I understand now and makes sense. Thanks @anvddriesch